### PR TITLE
[WFLY-7139] Handle "Component is stopped" exception better on undeploy for EJB client.

### PR DESCRIPTION
**Pull request by rachmatowicz**
This PR addresses: https://issues.jboss.org/browse/WFLY-7139
The downstream issue is: https://issues.jboss.org/browse/JBEAP-5764

### CODE REVIEW

File: ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package org.jboss.as.ee.component;
```
Comment (rhusar) [inline ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java]: Missing copyright header.

File: ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * Exception indicating that a component receiving a method call has stopped.
+ *
+ * @author Richard Achmatowicz
+ */
+public class ComponentIsStoppedException extends IllegalStateException {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Constructs a <tt>ComponentIsStoppedException</tt> with no detail message.
```
Comment (rhusar) [inline ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java]: IIUC the < tt > html element is deprecated, you can use < code > instead.

File: ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+     */
+    public ComponentIsStoppedException() {
+    }
+
+    /**
+     * Constructs a <tt>ComponentIsStoppedException</tt> with the specified
```
Comment (rhusar) [inline ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java]: Same as above.

**Review by rhusar — CHANGES_REQUESTED** [review]

**Comment by jfclere** [comment]
It is https://issues.jboss.org/browse/JBEAP-5764.

File: ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java (hunk 1 above)
Comment (rachmatowicz) [inline ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java]: Fixed.

File: ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java (hunk 2 above)
Comment (rachmatowicz) [inline ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java]: Fixed.

File: ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java (hunk 3 above)
Comment (rachmatowicz) [inline ee/src/main/java/org/jboss/as/ee/component/ComponentIsStoppedException.java]: Fixed.

**Comment by rhusar** [comment]
@jfclere more importantly, we need a downstream PR not just an acked EAP issue.

**Pull request merged**